jsonnet/kube-prometheus/kube-state-metrics/kube-state-metrics.libsonnet

```diff
@@ -310,6 +310,12 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
               scrapeTimeout: $._config.kubeStateMetrics.scrapeTimeout,
               honorLabels: true,
               bearerTokenFile: '/var/run/secrets/kubernetes.io/serviceaccount/token',
+              relabelings: [
+                {
+                  regex: '(pod|service|job|endpoint)',
```

Comment:
we want to also drop namespace here, no?

Reply:
No, thats the correct namespace, see in the example.